UnicornRender/video/source/vulkan/VkTexture.cpp

```diff
@@ -27,8 +27,9 @@ const vk::DescriptorImageInfo& VkTexture::GetDescriptorImageInfo() const
 }
 
 bool VkTexture::Create(const vk::PhysicalDevice& physicalDevice, const vk::Device& device,
-                       const vk::CommandPool& commandPool, const vk::Queue& queue, const Texture* texture)
+                       const vk::CommandPool& commandPool, const vk::Queue& queue, std::shared_ptr<Texture> texture)
 {
+    auto logger = spdlog::get(utility::Logger::LoggerName());
```

Comment:
`std::shared_ptr<spdlog::logger>` should be a class variable. There is no need to call `get()` every time.